Arguments: 
  /opt/homebrew/Cellar/node@18/18.16.1_1/bin/node /opt/homebrew/Cellar/yarn/1.22.19/libexec/bin/yarn.js add amplitude-experiment

PATH: 
  /opt/homebrew/opt/node@18/bin:/Users/tim.yiu/.pyenv/shims:/Users/tim.yiu/.pyenv/bin:/Users/tim.yiu/.pyenv/bin:/Users/tim.yiu/Library/Python/3.9/lib/python/site-packages/:/opt/homebrew/opt/postgresql@13/bin/:/opt/homebrew/opt/mysql@5.7/bin:/usr/local/opt/postgresql@13/bin/:/usr/local/opt/mysql@5.7/bin:/opt/homebrew/sbin:/opt/homebrew/bin:/usr/local/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/tim.yiu/bin:/Users/tim.yiu/.aws

Yarn version: 
  1.22.19

Node version: 
  18.16.1

Platform: 
  darwin arm64

Trace: 
  Error: https://registry.yarnpkg.com/amplitude-experiment: Not found
      at params.callback [as _callback] (/opt/homebrew/Cellar/yarn/1.22.19/libexec/lib/cli.js:66145:18)
      at self.callback (/opt/homebrew/Cellar/yarn/1.22.19/libexec/lib/cli.js:140890:22)
      at Request.emit (node:events:513:28)
      at Request.<anonymous> (/opt/homebrew/Cellar/yarn/1.22.19/libexec/lib/cli.js:141862:10)
      at Request.emit (node:events:513:28)
      at IncomingMessage.<anonymous> (/opt/homebrew/Cellar/yarn/1.22.19/libexec/lib/cli.js:141784:12)
      at Object.onceWrapper (node:events:627:28)
      at IncomingMessage.emit (node:events:525:35)
      at endReadableNT (node:internal/streams/readable:1359:12)
      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

npm manifest: 
  No manifest

yarn manifest: 
  No manifest

Lockfile: 
  # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
  # yarn lockfile v1
